packages/browser/scripts/source-map-cli.js

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env node
+
+const util = require('util');
+const path = require('path');
+const fs = require('fs');
+
+const { v4: uuidv4 } = require('uuid');
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+
+const lstat = util.promisify(fs.lstat);
+const readdir = util.promisify(fs.readdir);
+const readFile = util.promisify(fs.readFile);
+const appendFile = util.promisify(fs.appendFile);
+const writeFile = util.promisify(fs.writeFile);
+const exists = util.promisify(fs.exists);
+
+const INJECTOR_CODE =
+  '\n!function(){try{var e="undefined"!=typeof window?window:"undefined"!=typeof global?global:"undefined"!=typeof self?self:{},n=(new Error).stack;n&&(e._sentryDebugIds=e._sentryDebugIds||{},e._sentryDebugIds[n]="__DEBUG_ID__")}catch(e){}}()';
+
+const DEBUG_COMMENT_CODE = '\n//# sentryDebugId=__DEBUG_ID__';
```

Comment:
let's call this `__SENTRY_DEBUG_ID__` to stay on the safe side.

```suggestion
const DEBUG_COMMENT_CODE = '\n//# sentryDebugId=__DEBUG_ID__';
```